src/assets/_mixins.scss

```diff
@@ -0,0 +1,14 @@
+// FLEX
+
+@mixin jcc-aic {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin jcc-aic-h {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
```

Comment:
Thank you. You are right those are bit cryptic. Would use suggest staying away from mixins of this typ and maybe use generic classes like `.flex` that would do just `display: flex;` etc?

For now I will rename them for code readability.

Reply:
Personally I try to stray away from using mixins that just add some CSS (unless I use them in the same file; otherwise they might hide some small details which you might not see from a glance due to their globally imported nature) but it's an opinion and that's debatable 😄.
For example for media queries I absolutely go for mixins to standardize the screen sizes I use in them throughout the app.

However, if they are easily understood to what they do and you are comfortable using them then I guess it might be fine, I'd only consider the naming 😄

If you want some utility classes that's also a way to go of course, but a good practice might be to have some differentiation between utility and component classes, for example a prefix - `u-text-center`. This gives a hint to not look for this class in this component because it might be an utility that is most likely global